pom.xml

```diff
@@ -22,7 +22,7 @@
   <properties>
     <revision>1.36</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.3</jenkins.version>
+    <jenkins.version>2.220</jenkins.version>
```

Comment:
If we are considering merging this, there are other cleanups that should follow before releasing.

We can remove reflection for password, remove custom Proxy Configurator.
There is properly more stuff.